### DISCUSSION TITLE
Updates tab

### DISCUSSION THREAD

**Discussion by Cikapkamuy** [xgi/houdoku#133] [Feature Requests]
An updates tab similar to one that Tachiyomi has where it shows you new chapters for releasing manga that you haven't read yet and possibly a notification system to go with it.

**Comment by yash-278** [comment]
I believe as of right now, it would be unnecessary to add a notification tab, currently you can filter with chapters, keeping the Filter to Unread will remove all caught-up manga & only show the To-Read. It serves fairly the same purpose with the fact that not having additional logic for the new tab that will do the same thing, just in a different way.

I've been using Tachiyomi for years now, I can say that, the Updates tab, I haven't used it ever, if not mis-clicked sometimes 😂
The fact that I've loads of chapters to read, so It doesn't make sense to me to go and see which new chapter is released.
It might be added sometime in future, but probably not a priority, imo.

**Comment by Kinasho** [comment]
Yeah I don't see the merit for an updates tab, because well, it only refreshes your library on startup and manually. I'd assume that you'd see the manga that was released soon after opening the app. And well, with it only running when you open the app, how would there be a notification system when it won't refresh unless you open the app? I think it is better to leave it as is and just use a filter for unread, rather than having something idling in the background just to give you a notification, which it won't even do cause it won't refresh unless you open it.
